packages/cascara/src/atoms/Flex/Flex.js

```diff
@@ -43,14 +43,13 @@ const propTypes = {
 const Flex = ({
   as = 'div',
   children,
-  className,
   column = false,
   fluid = true,
   space = 'between',
   vAlign = 'start',
   ...rest
 }) => {
-  const classList = cx(className, {
+  const classList = cx(rest.className, {
     _: true,
```

Comment:
This underscore should not be part of the option object when it is always true. It should be the very first argument in the `cx()` function instead.

cc @lordpixel just as a heads up. Any class name in the options object that has `true` on it should be a separate argument, and it should usually be the first argument.

Reply:
`  const classList = cx('_', rest.className, {
    column: column,
    fluid: fluid,
    ...getClassOptions(spaceOptions, space, 'space'),
    ...getClassOptions(vAlignOptions, vAlign, 'vAlign'),
  });`